test.js

```diff
@@ -33,18 +33,22 @@ for (let i = 0; i < guides.length; i++) {
   });
 }
 
-const priceBoxes = document.querySelectorAll(".price-container");
+// target the menu cards not the price container since price container is being positioned
+const menuCards = document.querySelectorAll(".menu-card");
```

Comment:
targeting menu cards vs price-container since theres no positioning. The reason your click wasn't working the second time to toggle the class was because of positioning.